### ISSUE TITLE
reproduce results

### ISSUE THREAD

**Issue by jiag19** [BenfengXu/SSAN#7]
Hi there,

Thanks for your great work. I'm trying to reproduce the SSAN results based on roberta_large. My command is like below:

python ./run_docred.py   --model_type roberta   --entity_structure biaffine   --model_name_or_path ./pretrained_lm/roberta_large/   --do_train   --do_eval   --data_dir ./data/DocRED/   --max_seq_length 512   --max_ent_cnt 42   --per_gpu_train_batch_size 4   --learning_rate 3e-5   --num_train_epochs 40   --warmup_ratio 0.1   --output_dir checkpoints --seed 42   --logging_steps 10

But I got dev_F1 as 0.029, I was wondering if anything went wrong with my args. Could u help me check with this? Thanks.

**Comment by BenfengXu** [comment]
F1 = 0.029 is indeed a problematic results. You can: 
 - check your model ckpt, for your information, the md5sum checksum my downloaded roberta large pytorch model ckpt is `234a3b27e09d3486d7719c66ba1aaa31`
 - Roberta model can corrupt under particular settings in my previous training experimence, but not often, so you can switch to another different learning rate and the results will be normal, e.g., 2e-5, 4e-5, 5e-5.

**Comment by jiag19** [comment]
> F1 = 0.029 is indeed a problematic results. You can:
> 
> * check your model ckpt, for your information, the md5sum checksum my downloaded roberta large pytorch model ckpt is `234a3b27e09d3486d7719c66ba1aaa31`
> * Roberta model can corrupt under particular settings in my previous training experimence, but not often, so you can switch to another different learning rate and the results will be normal, e.g., 2e-5, 4e-5, 5e-5.

Thanks for your prompt reply. My md5 is the same as yours. I've tried 3e-5 and 5e-5 learning rates, both of them resulted in similarly low scores, I will see if other rates work. Thanks.

**Comment by BenfengXu** [comment]
Also, check if your dataset (`dev.json`) are named and loaded correctly.

And try to perform inference on train.json to check if model is actually corrupted.

And check the model's prediction, to see if its really 0.02 or just something in the eval process go wrong.

**Comment by xwjim** [comment]
I meet the same problem. Do you solve the problem?
